repos/spack_repo/builtin/packages/pbvr/package.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import platform
+import re
+
+from spack_repo.builtin.build_systems.makefile import MakefilePackage
+
+from llnl.util.filesystem import install_tree
+
+from spack.package import *
+from spack.util.environment import set_env
+
+
+class Pbvr(MakefilePackage):
+    """CS/IS-PBVR is a scientific visualization application designed
+    based on Particle-Based Volume Rendering (PBVR). This application
+    is capable of multivariate visualization and three-dimensional point cloud visualization
+    in addition to standard visualization functions such as volume rendering
+    and isosurfaces for 3D volume data obtained from simulations and measuring instruments.
+    In addition, the framework for distributed processing of optimized PBVR is characterized
+    by the ability to remotely visualize large-scale time-series volume data in remote locations
+    at high speed. As a method of remote visualization, you can choose between
+    client-server (CS) visualization, which visualizes volume data stored in remote storage,
+    and in-situ (IS) visualization, which visualizes simulations simultaneously and
+    in the same environment. This application is being developed at the Center
+    for Computational Science and e-Systems of the Japan Atomic Energy Agency."""
+
+    phases = ["build", "install"]
+
+    homepage = "https://github.com/CCSEPBVR/CS-IS-PBVR"
+    url = "https://github.com/CCSEPBVR/CS-IS-PBVR/archive/refs/tags/v3.5.0.tar.gz"
+
+    maintainers("sakamoto-naohito")
+
+    license("LGPL-3.0-only")
```

Comment:
Since this now vendors KVS, the license should reflect this:
```suggestion
    license("LGPL-3.0-only AND BSD-3-Clause")
```